### ISSUE TITLE
Migrate from B2D

### ISSUE THREAD

**Issue by ehazlett** [docker/machine#1329]
This will finalize the migration from boot2docker.  This is will migrate away from the b2d OS and to a similar more supported base OS.

**Comment by nathanleclaire** [comment]
Speaking of, have we accounted for the migration docs on https://github.com/docker/machine/blob/master/docs/b2d_migration.md to be included in the docs release for 0.3.0?  cc @moxiegirl

**Comment by moxiegirl** [comment]
@nathanleclaire @ehazlett My understanding from yesterday's meeting was that for 0.3.0 we **might** include a note that Boot2Docker is going to be deprecated. I think the migration documentation is premature. For 0.4.0 if we plan on asking people to migrate, there many more points in the documentation that need to be addressed.  I have a task for myself to scope the actual hits.

@samalba was taking the task to identify whether we would do the warning note in 0.3.0

If we want to change this plan and add a migration doc, I need to do some adjustments to my schedule to for the change. So, hopefully we can resolve this question today.

**Comment by moxiegirl** [comment]
@ehazlett @nathanleclaire @samalba I've talked to Sam about this.  Since machine is still in BETA for 1.7 and we don't want this to be an official migration until 1.8.0, I would recommend that we handle this like an experimental feature.   Those features  we don't document in the official documentation and risk confusing users.  Instead, the tack is this:

- create an experimental directory at the project root
- add an README.md of "experimental" features to set user expectations
- add experimental docs into that directory
- mention experimental features in the release notes and provide a link to the experimental README.md in the project

This is the solution we are using for [docker/docker](https://github.com/docker/docker/tree/master/experimental).  WDYT?

**Comment by ehazlett** [comment]
SGTM

**Comment by moxiegirl** [comment]
cool. Thanks Evan. ping @nathanleclaire

**Comment by nathanleclaire** [comment]
That sounds good to me.  If we are not going to include a comprehensive migration document, though, I want to at least include a section detailing the use of the `--virtualbox-import-boot2docker-vm` in `docs/index.md`.

**Comment by moxiegirl** [comment]
@nathanleclaire OK.  My scope is kind of limited tho. Were you thinking of this level of detail?

http://docs.docker.com/machine/#using-docker-machine-with-a-cloud-provider

Or something else? If you can rough out the command sequence, I can beef them up.  Normally, I'd just take time to do it all myself but I have too many critical path projects in the air at the moment.

**Comment by nathanleclaire** [comment]
@moxiegirl Yes, I hadn't envisioned anything fancy, just a subsection in the VirtualBox section labeled "Importing an existing boot2docker VM', with a sentence or two and a sample command like:

```console
$ docker-machine create -d virtualbox --virtualbox-import-boot2docker-vm boot2docker-vm b2d
```

**Comment by moxiegirl** [comment]
@nathanleclaire  Np. I can do that. I'll create a PR by EOD for the whole thing soup-to-nuts as we all discussed here.

**Comment by ehazlett** [comment]
@smashwilson can I ask why you closed this?  This issue is for the migration from b2d to machine (the larger issue not just docs).

**Comment by moxiegirl** [comment]
@ehazlett Closed because of: http://docs.docker.com/machine/migrate-to-machine/#migrate-from-boot2docker-to-docker-machine was done.  Also, all the work done last release for the Toolbox work.
